<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>MainWindow</class>
 <widget class="QMainWindow" name="MainWindow">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>490</width>
    <height>552</height>
   </rect>
  </property>
  <property name="focusPolicy">
   <enum>Qt::StrongFocus</enum>
  </property>
  <property name="windowTitle">
   <string>DMFB</string>
  </property>
  <property name="styleSheet">
   <string notr="true">QMainWindow
{
	background-color: rgb(67, 67, 67);
}

QMenuBar
{
	background-color: rgb(35, 35, 35);
	color: white;
	border-bottom: 2px solid rgb(50,50,50);
}

QToolBar
{
	background-color: rgb(35,35,35);
	border-bottom: 2px solid rgb(50,50,50);
}

QToolBar::separator
{
	background:rgb(50,50,50);
	width: 2px;
}

QToolBar::handle
{
	color: rgb(50,50,50);
}

</string>
  </property>
  <widget class="QWidget" name="centralWidget">
   <layout class="QGridLayout" name="gridLayout">
    <item row="0" column="1">
     <layout class="QHBoxLayout" name="horizontalLayout_2">
      <item>
       <spacer name="horizontalSpacer_7">
        <property name="orientation">
         <enum>Qt::Horizontal</enum>
        </property>
        <property name="sizeHint" stdset="0">
         <size>
          <width>40</width>
          <height>20</height>
         </size>
        </property>
       </spacer>
      </item>
      <item>
       <widget class="QLCDNumber" name="lcdNumber">
        <property name="sizePolicy">
         <sizepolicy hsizetype="Preferred" vsizetype="Preferred">
          <horstretch>0</horstretch>
          <verstretch>0</verstretch>
         </sizepolicy>
        </property>
        <property name="styleSheet">
         <string notr="true">font-size: 50px</string>
        </property>
        <property name="digitCount">
         <number>3</number>
        </property>
       </widget>
      </item>
      <item>
       <spacer name="horizontalSpacer_8">
        <property name="orientation">
         <enum>Qt::Horizontal</enum>
        </property>
        <property name="sizeHint" stdset="0">
         <size>
          <width>40</width>
          <height>20</height>
         </size>
        </property>
       </spacer>
      </item>
     </layout>
    </item>
    <item row="1" column="1">
     <widget class="DrawWidget" name="widget" native="true">
      <property name="sizePolicy">
       <sizepolicy hsizetype="Preferred" vsizetype="Preferred">
        <horstretch>2</horstretch>
        <verstretch>2</verstretch>
       </sizepolicy>
      </property>
      <property name="minimumSize">
       <size>
        <width>360</width>
        <height>300</height>
       </size>
      </property>
     </widget>
    </item>
    <item row="1" column="0">
     <spacer name="horizontalSpacer">
      <property name="orientation">
       <enum>Qt::Horizontal</enum>
      </property>
      <property name="sizeHint" stdset="0">
       <size>
        <width>59</width>
        <height>20</height>
       </size>
      </property>
     </spacer>
    </item>
    <item row="2" column="1">
     <layout class="QHBoxLayout" name="horizontalLayout">
      <item>
       <spacer name="horizontalSpacer_3">
        <property name="orientation">
         <enum>Qt::Horizontal</enum>
        </property>
        <property name="sizeHint" stdset="0">
         <size>
          <width>40</width>
          <height>20</height>
         </size>
        </property>
       </spacer>
      </item>
      <item>
       <widget class="QSlider" name="horizontalSlider">
        <property name="sizePolicy">
         <sizepolicy hsizetype="Fixed" vsizetype="Fixed">
          <horstretch>0</horstretch>
          <verstretch>0</verstretch>
         </sizepolicy>
        </property>
        <property name="focusPolicy">
         <enum>Qt::NoFocus</enum>
        </property>
        <property name="styleSheet">
         <string notr="true">QSlider::groove:horizontal
{
	height: 10px;
	left: 5px; 
 	right: 5px; 
	background: #dcdcdc;
}

QSlider::handle:horizontal
{
	border-radius: 10px;
	width:  20px;
	margin: -5px -1px;
	background: #2e2eff;
}
QSlider::sub-page:horizontal
{
	background: qlineargradient(spread:pad, x1:0, y1:1, x2:0, y2:0, stop:0 rgba(24, 44, 82, 255), stop:0.25 rgba(40, 73, 134, 255), stop:0.5 rgba(50, 80, 170, 255), stop:1 rgba(55, 125, 255, 255));
}</string>
        </property>
        <property name="value">
         <number>20</number>
        </property>
        <property name="orientation">
         <enum>Qt::Horizontal</enum>
        </property>
       </widget>
      </item>
      <item>
       <widget class="QPushButton" name="pushButton_last">
        <property name="focusPolicy">
         <enum>Qt::NoFocus</enum>
        </property>
        <property name="text">
         <string>last</string>
        </property>
       </widget>
      </item>
      <item>
       <spacer name="horizontalSpacer_4">
        <property name="orientation">
         <enum>Qt::Horizontal</enum>
        </property>
        <property name="sizeHint" stdset="0">
         <size>
          <width>40</width>
          <height>20</height>
         </size>
        </property>
       </spacer>
      </item>
      <item>
       <widget class="QPushButton" name="pushButton_play">
        <property name="focusPolicy">
         <enum>Qt::NoFocus</enum>
        </property>
        <property name="text">
         <string>play</string>
        </property>
       </widget>
      </item>
      <item>
       <spacer name="horizontalSpacer_5">
        <property name="orientation">
         <enum>Qt::Horizontal</enum>
        </property>
        <property name="sizeHint" stdset="0">
         <size>
          <width>40</width>
          <height>20</height>
         </size>
        </property>
       </spacer>
      </item>
      <item>
       <widget class="QPushButton" name="pushButton_next">
        <property name="focusPolicy">
         <enum>Qt::NoFocus</enum>
        </property>
        <property name="text">
         <string>next</string>
        </property>
       </widget>
      </item>
      <item>
       <spacer name="horizontalSpacer_6">
        <property name="orientation">
         <enum>Qt::Horizontal</enum>
        </property>
        <property name="sizeHint" stdset="0">
         <size>
          <width>40</width>
          <height>20</height>
         </size>
        </property>
       </spacer>
      </item>
      <item>
       <widget class="QPushButton" name="pushButton_reset">
        <property name="focusPolicy">
         <enum>Qt::NoFocus</enum>
        </property>
        <property name="text">
         <string>reset</string>
        </property>
       </widget>
      </item>
      <item>
       <spacer name="horizontalSpacer_11">
        <property name="orientation">
         <enum>Qt::Horizontal</enum>
        </property>
        <property name="sizeHint" stdset="0">
         <size>
          <width>40</width>
          <height>20</height>
         </size>
        </property>
       </spacer>
      </item>
     </layout>
    </item>
    <item row="1" column="2">
     <spacer name="horizontalSpacer_2">
      <property name="orientation">
       <enum>Qt::Horizontal</enum>
      </property>
      <property name="sizeHint" stdset="0">
       <size>
        <width>59</width>
        <height>20</height>
       </size>
      </property>
     </spacer>
    </item>
   </layout>
  </widget>
  <widget class="QMenuBar" name="menuBar">
   <property name="geometry">
    <rect>
     <x>0</x>
     <y>0</y>
     <width>490</width>
     <height>22</height>
    </rect>
   </property>
   <widget class="QMenu" name="menustart">
    <property name="title">
     <string>Start(&amp;S)</string>
    </property>
    <addaction name="actionLoad"/>
    <addaction name="actionOption"/>
    <addaction name="actionExit"/>
   </widget>
   <widget class="QMenu" name="menuRun">
    <property name="title">
     <string>Run(&amp;R)</string>
    </property>
    <addaction name="actionPlay"/>
    <addaction name="actionLast"/>
    <addaction name="actionNext"/>
    <addaction name="actionReset"/>
   </widget>
   <widget class="QMenu" name="menuHelp_H">
    <property name="title">
     <string>Help(&amp;H)</string>
    </property>
    <addaction name="actionInstruction"/>
    <addaction name="actionAbout"/>
   </widget>
   <addaction name="menustart"/>
   <addaction name="menuRun"/>
   <addaction name="menuHelp_H"/>
  </widget>
  <widget class="QToolBar" name="mainToolBar">
   <attribute name="toolBarArea">
    <enum>TopToolBarArea</enum>
   </attribute>
   <attribute name="toolBarBreak">
    <bool>false</bool>
   </attribute>
   <addaction name="actionLoad"/>
   <addaction name="actionOption"/>
   <addaction name="actionExit"/>
   <addaction name="separator"/>
   <addaction name="actionLast"/>
   <addaction name="actionPlay"/>
   <addaction name="actionNext"/>
   <addaction name="actionReset"/>
   <addaction name="separator"/>
   <addaction name="actionSound"/>
   <addaction name="actionInstruction"/>
   <addaction name="actionAbout"/>
   <addaction name="separator"/>
  </widget>
  <widget class="QStatusBar" name="statusBar"/>
  <action name="actionOption">
   <property name="icon">
    <iconset resource="png.qrc">
     <normaloff>:/png/option.png</normaloff>:/png/option.png</iconset>
   </property>
   <property name="text">
    <string>Option(&amp;O)</string>
   </property>
  </action>
  <action name="actionLoad">
   <property name="icon">
    <iconset resource="png.qrc">
     <normaloff>:/png/open-file.png</normaloff>:/png/open-file.png</iconset>
   </property>
   <property name="text">
    <string>Load(&amp;L)</string>
   </property>
  </action>
  <action name="actionExit">
   <property name="icon">
    <iconset resource="png.qrc">
     <normaloff>:/png/power-button.png</normaloff>:/png/power-button.png</iconset>
   </property>
   <property name="text">
    <string>Exit(&amp;E)</string>
   </property>
  </action>
  <action name="actionPlay">
   <property name="icon">
    <iconset resource="png.qrc">
     <normaloff>:/png/play.png</normaloff>:/png/play.png</iconset>
   </property>
   <property name="text">
    <string>Play(&amp;P)</string>
   </property>
  </action>
  <action name="actionLast">
   <property name="icon">
    <iconset resource="png.qrc">
     <normaloff>:/png/last.png</normaloff>:/png/last.png</iconset>
   </property>
   <property name="text">
    <string>Last(&amp;L)</string>
   </property>
  </action>
  <action name="actionNext">
   <property name="icon">
    <iconset resource="png.qrc">
     <normaloff>:/png/next.png</normaloff>:/png/next.png</iconset>
   </property>
   <property name="text">
    <string>Next(&amp;N)</string>
   </property>
  </action>
  <action name="actionReset">
   <property name="icon">
    <iconset resource="png.qrc">
     <normaloff>:/png/reset.png</normaloff>:/png/reset.png</iconset>
   </property>
   <property name="text">
    <string>Reset(&amp;R)</string>
   </property>
  </action>
  <action name="actionInstruction">
   <property name="icon">
    <iconset resource="png.qrc">
     <normaloff>:/png/instruction.png</normaloff>:/png/instruction.png</iconset>
   </property>
   <property name="text">
    <string>Instruction(&amp;I)</string>
   </property>
  </action>
  <action name="actionAbout">
   <property name="icon">
    <iconset resource="png.qrc">
     <normaloff>:/png/about.png</normaloff>:/png/about.png</iconset>
   </property>
   <property name="text">
    <string>About(&amp;A)</string>
   </property>
  </action>
  <action name="actionSound">
   <property name="icon">
    <iconset resource="png.qrc">
     <normaloff>:/png/sound.png</normaloff>:/png/sound.png</iconset>
   </property>
   <property name="text">
    <string>Sound</string>
   </property>
  </action>
 </widget>
 <layoutdefault spacing="6" margin="11"/>
 <customwidgets>
  <customwidget>
   <class>DrawWidget</class>
   <extends>QWidget</extends>
   <header location="global">drawwidget.h</header>
   <container>1</container>
  </customwidget>
 </customwidgets>
 <resources>
  <include location="png.qrc"/>
 </resources>
 <connections>
  <connection>
   <sender>actionOption</sender>
   <signal>triggered()</signal>
   <receiver>MainWindow</receiver>
   <slot>setOption()</slot>
   <hints>
    <hint type="sourcelabel">
     <x>-1</x>
     <y>-1</y>
    </hint>
    <hint type="destinationlabel">
     <x>244</x>
     <y>154</y>
    </hint>
   </hints>
  </connection>
  <connection>
   <sender>actionLoad</sender>
   <signal>triggered()</signal>
   <receiver>MainWindow</receiver>
   <slot>loadFile()</slot>
   <hints>
    <hint type="sourcelabel">
     <x>-1</x>
     <y>-1</y>
    </hint>
    <hint type="destinationlabel">
     <x>186</x>
     <y>222</y>
    </hint>
   </hints>
  </connection>
  <connection>
   <sender>actionExit</sender>
   <signal>triggered()</signal>
   <receiver>MainWindow</receiver>
   <slot>close()</slot>
   <hints>
    <hint type="sourcelabel">
     <x>-1</x>
     <y>-1</y>
    </hint>
    <hint type="destinationlabel">
     <x>186</x>
     <y>222</y>
    </hint>
   </hints>
  </connection>
  <connection>
   <sender>horizontalSlider</sender>
   <signal>valueChanged(int)</signal>
   <receiver>MainWindow</receiver>
   <slot>setInterval(int)</slot>
   <hints>
    <hint type="sourcelabel">
     <x>105</x>
     <y>516</y>
    </hint>
    <hint type="destinationlabel">
     <x>413</x>
     <y>500</y>
    </hint>
   </hints>
  </connection>
  <connection>
   <sender>actionInstruction</sender>
   <signal>triggered()</signal>
   <receiver>MainWindow</receiver>
   <slot>showInstruction()</slot>
   <hints>
    <hint type="sourcelabel">
     <x>-1</x>
     <y>-1</y>
    </hint>
    <hint type="destinationlabel">
     <x>244</x>
     <y>275</y>
    </hint>
   </hints>
  </connection>
  <connection>
   <sender>actionAbout</sender>
   <signal>triggered()</signal>
   <receiver>MainWindow</receiver>
   <slot>showAbout()</slot>
   <hints>
    <hint type="sourcelabel">
     <x>-1</x>
     <y>-1</y>
    </hint>
    <hint type="destinationlabel">
     <x>244</x>
     <y>275</y>
    </hint>
   </hints>
  </connection>
  <connection>
   <sender>actionLast</sender>
   <signal>triggered()</signal>
   <receiver>pushButton_last</receiver>
   <slot>click()</slot>
   <hints>
    <hint type="sourcelabel">
     <x>-1</x>
     <y>-1</y>
    </hint>
    <hint type="destinationlabel">
     <x>151</x>
     <y>509</y>
    </hint>
   </hints>
  </connection>
  <connection>
   <sender>actionNext</sender>
   <signal>triggered()</signal>
   <receiver>pushButton_next</receiver>
   <slot>click()</slot>
   <hints>
    <hint type="sourcelabel">
     <x>-1</x>
     <y>-1</y>
    </hint>
    <hint type="destinationlabel">
     <x>335</x>
     <y>509</y>
    </hint>
   </hints>
  </connection>
  <connection>
   <sender>actionPlay</sender>
   <signal>triggered()</signal>
   <receiver>pushButton_play</receiver>
   <slot>click()</slot>
   <hints>
    <hint type="sourcelabel">
     <x>-1</x>
     <y>-1</y>
    </hint>
    <hint type="destinationlabel">
     <x>243</x>
     <y>509</y>
    </hint>
   </hints>
  </connection>
  <connection>
   <sender>actionReset</sender>
   <signal>triggered()</signal>
   <receiver>pushButton_reset</receiver>
   <slot>click()</slot>
   <hints>
    <hint type="sourcelabel">
     <x>-1</x>
     <y>-1</y>
    </hint>
    <hint type="destinationlabel">
     <x>427</x>
     <y>509</y>
    </hint>
   </hints>
  </connection>
  <connection>
   <sender>actionSound</sender>
   <signal>triggered()</signal>
   <receiver>MainWindow</receiver>
   <slot>sound()</slot>
   <hints>
    <hint type="sourcelabel">
     <x>-1</x>
     <y>-1</y>
    </hint>
    <hint type="destinationlabel">
     <x>244</x>
     <y>275</y>
    </hint>
   </hints>
  </connection>
 </connections>
 <slots>
  <slot>setOption()</slot>
  <slot>loadFile()</slot>
  <slot>setInterval(int)</slot>
  <slot>reset()</slot>
  <slot>showAbout()</slot>
  <slot>showInstruction()</slot>
  <slot>sound()</slot>
 </slots>
</ui>
